.github/FUNDING.yml

```
@@ -0,0 +1,3 @@
+github: Exodus-Privacy
+liberapay: exodus
+custom: "https://exodus-privacy.eu.org/en/page/contribute/"
```

```suggestion
custom: "https://exodus-privacy.eu.org/en/page/contribute/"

```

Missing end of file line